### ISSUE TITLE
Bump python-kafka to 1.3.1

### ISSUE THREAD

**Issue by Drizzt1991** [aio-libs/aiokafka#58]
There are a few broken tests on this version

```
    @classmethod
    def encode(cls, items):
        # RecordAccumulator encodes messagesets internally
        if isinstance(items, io.BytesIO):
            size = Int32.decode(items)
            # rewind and return all the bytes
            items.seek(-4, 1)
            return items.read(size + 4)

        encoded_values = []
>       for (offset, message) in items:
E       ValueError: too many values to unpack (expected 2)

```

```
>       self.assertEqual(sorted(brokers), sorted(list(c_brokers)))
E       AssertionError: Lists differ: [(0, 'broker_1', 4567), (1, 'broker_2', 5678)] != [BrokerMetadata(nodeId=0, host='broker_1',[83 chars]one)]
E       
E       First differing element 0:
E       (0, 'broker_1', 4567)
E       BrokerMetadata(nodeId=0, host='broker_1', port=4567, rack=None)
E       
E       - [(0, 'broker_1', 4567), (1, 'broker_2', 5678)]
E       + [BrokerMetadata(nodeId=0, host='broker_1', port=4567, rack=None),
E       +  BrokerMetadata(nodeId=1, host='broker_2', port=5678, rack=None)]
```

**Comment by rmuslimov** [comment]
+1 here. kafka-python 1.3.1 has important bugfixes. However, aiokafka is not working with latest kafka-python.

**Comment by Drizzt1991** [comment]
Fixed in #60
